compiler/tests/lowering/fixtures/base_tuple_lowered.fe

```diff
@@ -0,0 +1,68 @@
+struct tuple_u256_bool:
```

Comment:
Regarding the naming collisions. We could suffix all of them with an uuid and call it a day (e.g. `tuple_u256_bool_ce6819ea_e0a1_4af2_92f5_7d3be21437d1`). Alternatively I guess we could try to generate the most humand readable name possible (so, `tuple_u256_bool`) **unless** we detect that the user has also used the same name in which case we just append something like `_1`. In any case, we should create an issue for that as it should have a high priority.

Reply:
Btw, super cool that we have these lowering tests now!

Reply:
I'd prefer to have a solution that does not involve checking to see if a name is already take. Something to discuss in the issue once it's written..